cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js

```diff
@@ -79,9 +71,13 @@ export const deleteProfile = (namespace, profile, currentNamespace) => {
     namespace,
     profile
   });
-  deleteObservable.subscribe(() => {
-    getProfiles(currentNamespace);
-  });
+  deleteObservable.subscribe(
+    () => {
+      getProfiles(currentNamespace);
+    },
+    (err) => {
+      Observable.throw(err);
```

Comment:
This would result in a js error right?

Reply:
It doesn't, you can track the execution if try to delete an enabled profile from the profiles list view. Do you mean it should be `return Observable.throw(err);`?